Add link to messages app

Gal-f opened the pull request:
Added link to messages HTML page from navbar
Added HTML page to urls.py

Fix #85

Gal-f committed:
Link Messages app to main page and address paths

Altering default/page.html to add links to Messages app
Adding Messages app addresses to urls.py

This commit also rebases the branch to upstream/main.

Omeramsc commented:
Please rebase and fix the conflicts, 
besides that, LGTM.

Omeramsc commented:
@Gal-f  it has new conflicts, please fix them.

Gal-f commented:
@Omeramsc solved, thanks for the heads-up.

ravidbro commented:
please rebase this branch on upstream main

Gal-f commented:
> please rebase this branch on upstream main

Done.
Can you please explain why this should be done? What is this different than solving the conflicts and sending a merge commit in GitHub? 🙏  @ravidbro

ravidbro commented on StuDash/urls.py:
```diff
@@ -11,6 +12,8 @@
     path("", IndexPageView.as_view(), name="home"),
     path("bookmarks/", include("bookmarks.urls"), name="bookmarks"),
     path("grades/", include("grades.urls"), name="grades"),
+    path("messages", ViewMessages, name="Messages"),
```
I see that the existing urls have trailing '/'
don't you want to be consistent? (or maybe you need it too)

Gal-f replied:
Figured I should, though since I wasn't sure what's the reason for that, I thought it better to leave it the way I understand and if needed, change to the '/' version in another PR with an explanation why I'm doing this.

ravidbro replied:
also, the capital letter on 'name' is not consistent

ravidbro replied:
But is it working like that?
I guess we will know only after the views will be merged into main

Gal-f replied:
Exactly what I thought. If it's not a complete mistake, I planned to merge it like this and if needed, improve it when I understand why.

ravidbro replied:
Consistent code within a project is important. First, add a code that looks like all the existing code,
If you think some other way is better, then you send another PR that fixes all the code (yours and others).
alternatively, you can fix their first and then add yours

ravidbro replied:
same for the '/'
consistent within the project

Gal-f replied:
Gotcha. Changed it and added an app-specific 'urls.py' accordingly, which can probably do more than it does now. I'll look into it further and adjust it accordingly (in a new dedicated PR).

ravidbro commented:
> > please rebase this branch on upstream main
> 
> Done.
> Can you please explain why this should be done? What is this different than solving the conflicts and sending a merge commit in GitHub?  @ravidbro

I saw that you merged from main to your branch, usually, you should do the opposite - rebase your branch on main and only at the end merge from your branch to main, but merging from main to your branch makes the log complicated

Gal-f committed:
Altered link in urls.py + added inner urls file

Changed Studash/urls.py messages app line to be consistent with trailing
'/' and linking to an app-specific urls.py file.
